articles/storage/blobs/lifecycle-management-policy-configure.md

```diff
@@ -210,6 +210,8 @@ To define a lifecycle management policy with an Azure Resource Manager template,
 ---
 
 A lifecycle management policy must be read or written in full. Partial updates are not supported.
+> [!NOTE]
+> Each rule can have up to 10 case-sensitive prefixes and upto 10 blob index tag conditions.
```

Comment:
Each rule can have up to ten case-sensitive prefixes, as well as up to ten blob index tag conditions.

Reply:
Hi @tamram -Each rule can have up to ten case-sensitive prefixes, as well as up to ten blob index tag conditions- This looks good. I accept these, kindly proceed with merge if it's okay.